pkg/gecko_sdk/Makefile

```diff
@@ -7,6 +7,8 @@ ifneq ($(CPU),efm32)
   $(error This package can only be used with EFM32 CPUs)
 endif
 
+CFLAGS += -Wno-error=implicit-fallthrough
```

Comment:
minor nit: we typically use a different notation: `-Wno-implicit-fallthrough`

Reply:
I would prefer if the error is disabled rather than hiding the warning entirely.

Reply:
good point, make sense to retain the warning

Reply:
This is even more important in this package, as the upstream software is mostly used in RIOT. So if we hide the warning in RIOT, this will never be fixed upstream.

Reply:
agreed

Reply:
on the other hand: murdock doesn't like it, at least there is an error.

Reply:
Compiler too old?

Reply:
well we do have a range of different, platform dependent compilers of various version - hence we need a solution that works for a considerable number of them. That means, GCC 5, 6, and 7 for instance; and without much if-else stuff.

I agree it would be best to have the warnings but if that doesn't work we should fall-back to use `-Wno-implicit-fallthrough` as we do for in (many) other cases.